tests/models/test_base.py

```diff
@@ -461,16 +342,69 @@ def test_full_model_works_for_all_without_features(self, kind: str) -> None:
     @pytest.mark.parametrize("dataset_key", ("no_features", "with_features"))
     @pytest.mark.parametrize("kind", ("u2i", "i2i"))
     @pytest.mark.parametrize("model_key", ("hot", "hot_warm"))
-    def test_not_cold_models_raise_on_cold(self, dataset_key: str, kind: str, model_key: str) -> None:
-        targets = self.colds[kind]
+    def test_not_cold_models_with_cold_targets(self, dataset_key: str, kind: str, model_key: str) -> None:
+        targets = self.colds[kind] + self.hots[kind]
+
+        # raise
         with pytest.raises(ValueError, match="doesn't support recommendations for cold"):
-            self._get_reco(targets, model_key, dataset_key, kind)
+            self._get_reco(targets, model_key, dataset_key, kind, on_unsupported_targets="raise")
+
+        # ignore
```

Comment:
Please NEVER test multiple cases in one test.  Ask me if you want me to explain why.    
Please, either write multiple tests, or use `parametrize` or `subtest`

Reply:
Fixed